PlayedTracksFetcher.cs

```diff
@@ -0,0 +1,130 @@
+using Microsoft.Azure.Functions.Worker;
+
+namespace Synk.Spotify;
+
+public class PlayedTracksFetcher
+{
+    private readonly TokenStore tokenStore;
+    private readonly UserStore userStore;
+    private readonly MusicStore musicStore;
+    private readonly PlayedTracksStore playedTracksStore;
+    private readonly TokenRefresher tokenRefresher;
+    private readonly SpotifyApi api;
+
+    public PlayedTracksFetcher(
+        TokenStore tokenStore,
+        UserStore userStore,
+        MusicStore musicStore,
+        PlayedTracksStore playedTracksStore,
+        TokenRefresher tokenRefresher,
+        SpotifyApi api)
+    {
+        this.tokenStore = tokenStore;
+        this.userStore = userStore;
+        this.musicStore = musicStore;
+        this.playedTracksStore = playedTracksStore;
+        this.tokenRefresher = tokenRefresher;
+        this.api = api;
+    }
+
+    [Function(nameof(PlayedTracksFetcher))]
```

Comment:
What will this result in in terms of URL? I.e. we have synk.azurewebsites.net, what will be after the slash?

Reply:
I am not sure it will be reachable through an endpoint when it has a TimerTrigger